packages/next-codemod/bin/upgrade.ts

```diff
@@ -47,17 +47,14 @@ export async function runUpgrade(): Promise<void> {
   }
   let targetVersionSpecifier: VersionSpecifier = ''
 
-  const shortcutVersion = process.argv[2]?.replace('@', '')
-  if (shortcutVersion) {
-    const res = await fetch(
-      `https://registry.npmjs.org/next/${shortcutVersion}`
-    )
+  if (distTag !== undefined) {
+    const res = await fetch(`https://registry.npmjs.org/next/${distTag}`)
     if (res.status === 200) {
       targetNextPackageJson = await res.json()
       targetVersionSpecifier = targetNextPackageJson.version
     } else {
       console.error(
-        `${chalk.yellow('Next.js ' + shortcutVersion)} does not exist. Check available versions at ${chalk.underline('https://www.npmjs.com/package/next?activeTab=versions')}, or choose one from below\n`
+        `${chalk.yellow(`next@distTag`)} does not exist. Check available versions at ${chalk.underline('https://www.npmjs.com/package/next?activeTab=versions')}, or choose one from below\n`
```

Comment:
`<tag>` looks more like a tag argument? to distinguish from `next@latest` `next@canary`. Thye might think `distTag` is a tag
```suggestion
        `${chalk.yellow(`next@<tag>`)} does not exist. Check available versions at ${chalk.underline('https://www.npmjs.com/package/next?activeTab=versions')}, or choose one from below\n`
```

Reply:
> <tag> looks more like a tag argument? to distinguish from next@latest next@canary. Thye might think distTag is a tag

dist-tag is actually what NPM calls them: https://docs.npmjs.com/adding-dist-tags-to-packages

But I'm going with `revision` instead. I need some word that encapsulates dist tags as well as exact versions.